pkg/antctl/runtime/runtime.go

```diff
@@ -53,6 +56,15 @@ func init() {
 	podName, found := os.LookupEnv("POD_NAME")
 	InPod = found && (strings.HasPrefix(podName, "antrea-agent") || strings.HasPrefix(podName, "antrea-controller") ||
 		strings.HasPrefix(podName, "flow-aggregator"))
+
+	if runtime.IsWindowsPlatform() && !InPod {
```

Comment:
I remember the default mode is ControllerMode if no environment variable settings, this design supports antctl on a remote client other than K8s Node. I doubt change may cause antctl fails to run on a remote Windows client?

Reply:
There should be no token on a remote client.
The only issue is if you want antctl controller mode on a Windows K8s Node, which doesn't seem like something someone would typically do.